backend/src/apiserver/storage/db_status_store.go

```diff
@@ -52,15 +52,16 @@ func (s *DBStatusStore) InitializeDBStatusTable() error {
 
 	// The table is not initialized
 	if !rows.Next() {
-		sql, args, err := sq.
+		sql, args, queryErr := sq.
```

Comment:
why only this place but not others similiar places?

Reply:
This is the only one go vet complains about since it's a redeclaration of err due to the other variables on the same line.